src/main/java/reposense/authorship/analyzer/AnnotatorAnalyzer.java

```diff
@@ -45,6 +45,9 @@ public static void aggregateAnnotationAuthorInfo(FileInfo fileInfo, Map<String,
         }
     }
 
+    /**
+     * Find the author that authors a line
```

Comment:
`Extracts the author name from the given {@code line}, finds the corresponding {@code Author} in {@code authorAliasMap}, and returns the result.`